comps/comps-foreman-plugins-fedora24.xml

```diff
@@ -71,11 +73,16 @@
       <packagereq type="default">rubygem-deface</packagereq>
       <packagereq type="default">rubygem-docker-api</packagereq>
       <packagereq type="default">rubygem-dynflow</packagereq>
+      <packagereq type="default">rubygem-faraday_middleware</packagereq>
+      <packagereq type="default">rubygem-fog-azure</packagereq>
       <packagereq type="default">rubygem-graphite-api</packagereq>
       <packagereq type="default">rubygem-hammer_cli_foreman_ssh</packagereq>
       <packagereq type="default">rubygem-logify</packagereq>
+      <packagereq type="default">rubygem-macaddr</packagereq>
```

Comment:
This can be removed, it's in Fedora (https://apps.fedoraproject.org/packages/rubygem-macaddr).